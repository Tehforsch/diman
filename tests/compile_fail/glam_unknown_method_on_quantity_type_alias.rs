#![allow(incomplete_features)]
#![feature(generic_const_exprs, adt_const_params)]
pub mod example_system;

fn main() {
    use example_system::vec2::Length;
    Length::unknown_method(49.0);
}
