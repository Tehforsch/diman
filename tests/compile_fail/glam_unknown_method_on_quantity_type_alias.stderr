error[E0599]: no function or associated item named `unknown_method` found for struct `example_system::Quantity` in the current scope
 --> tests/compile_fail/glam_unknown_method_on_quantity_type_alias.rs:7:13
  |
7 |     Length::unknown_method(49.0);
  |             ^^^^^^^^^^^^^^ function or associated item not found in `Quantity<Vec2, Dimension>`
  |
 ::: tests/compile_fail/example_system/mod.rs
  |
  |     Quantity,
  |     -------- function or associated item `unknown_method` not found for this struct
